connector/connect/src/main/protobuf/spark/connect/relations.proto

```diff
@@ -213,7 +213,7 @@ message Deduplicate {
 
 message LocalRelation {
   repeated Expression.QualifiedAttribute attributes = 1;
-  // TODO: support local data.
+  repeated bytes data = 2;
```

Comment:
I'm not sure this needs to be `repeated bytes` here because `bytes` itself is binary data of "arbitrary" length.

Reply:
+1

Reply:
Thank you, I use `repeated bytes`  in case that the batch size is lager than maxRecordsPerBatch,  I think is enough to use `bytes` here since `LocalRelation` is mostly used in debugging cases.